src/routes/Home/components/Repo/theme.css

```diff
@@ -122,7 +121,7 @@
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  font-family: 'Roboto', sans-serif;
+  font-family: var(--pullp-font);
```

Comment:
This didn't have `text-transform: uppercase;` on it, but will now be receiving that from the font variable. So either need to override it in here or make a different version of Roboto that isn't uppercased. Maybe just overriding is easier since 95% of the time Roboto is being uppercased in the app.